src/main/io/ledstrip.h

```diff
@@ -30,12 +30,12 @@
 
 #define TASK_LEDSTRIP_RATE_HZ 60
 
-#define LED_CONFIGURABLE_COLOR_COUNT   16
-#define LED_MODE_COUNT                  6
-#define LED_DIRECTION_COUNT             6
-#define LED_BASEFUNCTION_COUNT          10
-#define LED_OVERLAY_COUNT               7
-#define LED_SPECIAL_COLOR_COUNT        11
+#define LED_CONFIGURABLE_COLOR_COUNT     16
+#define LED_FCSTATE_COUNT                6
+#define LED_DIRECTION_COUNT               6
+#define LED_BASEFUNCTION_COUNT           10
+#define LED_OVERLAY_COUNT                 7
+#define LED_SPECIAL_FCSTATE_COLOR_COUNT 11
```

Comment:
```suggestion
#define LED_FCSTATE_SPECIAL_COLOR_COUNT 11
```

Reply:
makes sense; such would match this code:
```
src/main/io/ledstrip.c:1333:    } else if (fcStatusIndex == LED_FCSTATE_SPECIAL) {
src/main/io/ledstrip.h:101:    LED_FCSTATE_SPECIAL,
src/main/cli/cli.c:2039:            cliDefaultPrintLinef(dumpMask, equalsDefault, format, LED_FCSTATE_SPECIAL, j, colorIndexDefault);
src/main/cli/cli.c:2041:        cliDumpPrintLinef(dumpMask, equalsDefault, format, LED_FCSTATE_SPECIAL, j, colorIndex);
```
so modifying it, would need to change here as well:
```
src/main/io/ledstrip.c:1334:        if (modeColorIndex < 0 || modeColorIndex >= LED_SPECIAL_FCSTATE_COLOR_COUNT)
src/main/io/ledstrip.h:38:#define LED_SPECIAL_FCSTATE_COLOR_COUNT 11
src/main/io/ledstrip.h:162:    uint8_t color[LED_SPECIAL_FCSTATE_COLOR_COUNT];
src/main/msp/msp.c:1758:        for (int j = 0; j < LED_SPECIAL_FCSTATE_COLOR_COUNT; j++) {
src/main/cli/cli.c:2032:    for (uint32_t j = 0; j < LED_SPECIAL_FCSTATE_COLOR_COUNT; j++) {
```

Reply:
changed all instances, thanks